frontend/src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import { ThemeProvider } from "styled-components";
+
+import GlobalStyles from "./components/styles/Global";
+import { Theme }  from "./components/styles/Theme"
+
+import RecipePage from "./components/RecipePage";
+import Navbar from "./components/Navbar.jsx";
+import AboutMe from "./components/About.jsx";
+import Home from "./components/Home";
+
+function App() {
+  let [recipes, setRecipes] = useState();
+
+  useEffect(() => {
+    const getRecipes = () => {
+      fetch("http://localhost:3001/api/recipe/")
+        .then((res) => res.json())
+        .then((recipeData) => setRecipes(recipeData))
+        .catch((err) => {
+          throw new Error(err);
+        });
+    };
+    getRecipes();
+  }, []);
+
+  return (
+    <ThemeProvider theme={Theme}>
```

Comment:
It looks like you got some practice using Styled Components! This is great experience, you will likely see these again in industry if you work with React!